tools/podmanhtmltest.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+set -ex
+
+podman run --rm -ti \
```

Comment:
would be nice to have docker as a tool as well
(for example on macos I don't have a working podman)

Reply:
Done.